docs/release-notes/5.7.40-43.md

```diff
@@ -0,0 +1,64 @@
+
+<style>
+    table {
+        width=50%';
+        font-family: Chivo, Colfax, Franziska, Helvetica, Arial, sans-serif;
+    }
+    table td {
+        border: 0px;
+        padding: 8px;
+    }
+    tr:nth-child(even){
+        background-color:#f5f5f5
+    }
+    tr:hover {
+        background-color: #dddd;
+    }
+</style>
+
+| Release date | November 2022 |
+|---|---|
+| Install instructions | [Install Percona Server for MySQL](https://www.percona.com/doc/percona-server/5.7/installation.html) |
+| Download this version | [Percona Server for MySQL](https://www.percona.com/downloads/Percona-Server-5.7/LATEST/)
+
+[Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) 5.7.40-43
+includes all the features and bug fixes available in [MySQL 5.7.40 Community Edition](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-40.html) in addition to enterprise-grade features developed by Percona.
+
+*Percona Server for MySQL* is a free, fully compatible, enhanced, and open
+source drop-in replacement for any *MySQL* database. It provides superior
+performance, scalability, and instrumentation.
+
+*Percona Server for MySQL* is trusted by thousands of enterprises to provide
+better performance and concurrency for their most demanding workloads. It
+delivers more value to *MySQL* server users with optimized performance,
+greater performance scalability and availability, enhanced backups, and
+increased visibility.
+
+For paid [support](https://www.percona.com/services/support), [managed services](https://www.percona.com/services/managed-services) or [consulting services](https://www.percona.com/services/consulting), contact [Percona Sales](https://www.percona.com/about-percona/contact)
+
+For [training](https://www.percona.com/training), contact [Percona Training - Start learning now](https://learn.percona.com/contact-me)
+
+## Release highlights
+
+Improvements and bug fixes provided by Oracle for *MySQL* 5.7.40 and included in Percona Server for MySQL are the following:
+
+* ISO 8601 timestamps in log messages did not consider daylight saving time when `--log-timestamps=SYSTEM` was used.
+
+* The `GRANT OPTION` privilege was treated as related to database operations.
+
+* In specific cases, a `TRUNCATE TABLE` operation failed to release an acquired mutex.
+
+* A descending b-tree scan raised a debug assertion failure in debug builds.
+
+Find the full list of bug fixes and changes in the [MySQL 5.7.40 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-40.html).
+
+## Bug fixes
+
+* [PS-1098](https://jira.percona.com/browse/PS-1098): The incorrect log was used when using `audit_log_flush` more than once to rotate the logs manually.
+
+## Useful links
+
+The [*Percona Server for MySQL* GitHub location](https://github.com/percona/percona-server)
+
+[Contribute to the documentation](https://github.com/percona/percona-server/blob/8.0/doc/source/contributing.md)
```

Comment:
should this point to the contributing.md file in psmysql-docs?

Reply:
Fixed